addons/stock/views/stock_picking_views.xml

```diff
@@ -263,6 +264,10 @@
                     <field name="state" widget="statusbar" statusbar_visible="draft,confirmed,assigned,done" />
                     <button name="action_cancel" attrs="{'invisible': [('state', 'not in', ('assigned', 'confirmed', 'draft', 'waiting'))]}" string="Cancel" groups="base.group_user" type="object" data-hotkey="z"/>
                 </header>
+                <div class="alert alert-warning mb-0" role="alert" attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('partner_credit_warning', '=', False)]}">
+                    This customer has reached his Credit Limit of : <b><field name="partner_credit_limit"/></b>.<br/>
+                    Total amount due : <b><field name="partner_credit"/></b>.
+                </div>
```

Comment:
IMO this is a strange thing to show in a picking. I would think a warehouse worker/manager probably doesn't care about the delivery recipient's outstanding invoice amount. It should be confirmed by @thomasdeleval if we really want this behavior.

Reply:
Indeed I don't think it's a good idea, I think it's the salesman responsibility to decide that by confirming or not the SO.
I see some use cases for delivery but it think it's annecdotic

Reply:
OK then, I'll remove the warning from the Delivery Orders :ok_hand:

Reply:
We were pondering if that was useful or not ; we saw some feedback asking for it. But you guys have the last word and this feature can be safely removed from delivery orders.